src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java

```diff
@@ -1324,9 +1328,10 @@ public void onPrefsLoaded(SharedPreferences preferences) {
     }
 
     /* package */ UpdatesFromMixpanel constructUpdatesFromMixpanel(final Context context, final String token) {
-        if (Build.VERSION.SDK_INT < MPConfig.UI_FEATURES_MIN_API) {
-            Log.i(LOGTAG, "Web Configuration, A/B Testing, and Dynamic Tweaks are not supported on this Android OS Version");
-            return new UnsupportedUpdatesFromMixpanel(sSharedTweaks);
+        if (Build.VERSION.SDK_INT < MPConfig.UI_FEATURES_MIN_API
```

Comment:
Fair enough. We should have a proper log message in that case. SDK version is not supported VS manually disabled.